test_helm_install_uninstall.py

```diff
@@ -0,0 +1,140 @@
+import pytest
+import subprocess
+from time import sleep
+import hpe_3par_kubernetes_manager as manager
+import logging
+
+timeout = 900
+
+def test_helm_uninstall():
+    try:
+        node_obj = manager.hpe_list_node_objects()
+        node_list = node_obj.items
+        # Fetching master node ip
+        for item in node_list:
+            if item.spec.taints != "None":
+                host_ip = item.status.addresses[0].address
+                host_name = item.status.addresses[1].address
+            break
+
+        # ssh to master node and check csi driver installed
+        command = "helm ls -n kube-system"
+        command_output = manager.get_command_output(host_name, command)
+        if len(command_output) == 1:
+            print("Driver not installed")
+            return
+
+        # ssh to master node and execute uninstall commands
+        command = "helm uninstall hpe-csi --namespace kube-system"
+        command_output = manager.get_command_output(host_name, command)
+        assert command_output[0] == 'release "hpe-csi" uninstalled', "Uninstall of 'hpe-csi' driver failed"
+        sleep(30)
+
+        # Delete crds
+        crd_obj = manager.hpe_list_crds()
+        for crd in crd_obj:
+            manager.hpe_delete_crd(crd)   
+        sleep(30) 
+        crd_obj = manager.hpe_list_crds()
+        assert len(crd_obj) ==0,"Failed crd deletion"
+
+        # Check plugin pods have been deleted
+        pod_obj = manager.hpe_list_pod_objects("kube-system")
+        pod_list = pod_obj.items
+        for item in pod_list:
+            if 'app' in item.metadata.labels:
+                if item.metadata.labels['app']=='hpe-csi-controller' or item.metadata.labels['app']=='hpe-csi-node' or item.metadata.labels['app']=='primera3par-csp':
+                    print("plugin pod not deleted {0}. Status of the pod {1}".format(item.metadata.name, item.status.phase)) 
+            else:
+                continue
+        print("Plugin pods deleted")        
```

Comment:
Same as previous comment. Suggested to use polling.

Reply:
Addressed comment